plenario/templates/admin/view-datasets.html

```diff
@@ -150,6 +154,11 @@
                                 class="btn btn-success btn-sm">
                                 <i class='fa fa-check'></i> Approve
                             </a>
+                        {% else %}
+                            <a href="{{ url_for('views.update_dataset_view', source_url_hash=dataset.source_url_hash) }}"
+                               class="btn btn-info btn-sm">
+                                <i class='fa fa-navicon'></i> Update
+                            </a>
```

Comment:
Added a button that lets admins fire off update tasks manually, instead of waiting for a cron task (helpful for testing)